specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/examples/IntegrationRuntimeNodes_Update.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
'In operation "IntegrationRuntimeNodes_Update", parameter updateIntegrationRuntimeNodeRequest is required in the swagger spec but is not present in the provided example parameter values.'

Reply:
Fixed.